SekoiaIO/endpoint/_meta/smart-descriptions.json

```diff
@@ -1460,5 +1460,51 @@
                 "value": "SecurityCenter"
             }
         ]
+    },
+    {
+        "value": "Remote thread created from {process.executable} on {host.hostname}",
+        "conditions": [
+            {
+                "field": "event.code",
+                "value": "8"
+            },
+            {
+                "field": "event.provider",
+                "value": "SEKOIA-IO-Endpoint"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "process.command_line",
+                "target": "host.hostname",
+                "type": "executed on"
+            }
+        ]
```

Comment:
Ideally, we should also have a relationship 

source_process created remote thread in target_process

source_process and target_process fields should also have an observable mapping so that they appear on the graph